src/wms/src/main/java/org/geoserver/wms/kvp/FilterKvpParser.java

```diff
@@ -75,12 +75,28 @@ public Object parse(String value) throws Exception {
                 Filter filter;
                 final byte[] rawContent = string.getBytes();
                 InputStream input = new ByteArrayInputStream(rawContent);
-
+                if (string.contains("<fes:")) {
```

Comment:
this is not a good check; a user is free to use a different alias for the namespace

Reply:
Can you please reverify now?